mentat/llm_api.py

```diff
@@ -112,7 +122,7 @@ def choose_model(messages: list[dict[str, str]], allow_32k) -> str:
 
 @dataclass
 class CostTracker:
-    total_cost: int = 0
+    total_cost: float = 0
```

Comment:
I'd prefer to keep track of money as cents so it can be an int. Not super critical since we're not using the number to bill.

Reply:
The way we calculate it is by dividing the tokens by 1000 and multiplying by the cost, so no matter what we do it will always be a float; definitely in any scenario that wasn't a rough estimate I agree that money should always be an int though!